src/pint/event_toas.py

```diff
@@ -145,6 +152,10 @@ def load_fits_TOAs(
         Force this time system
     timeref : str, default None
         Forse this time reference
+    minmjd : float, default 0
```

Comment:
This is really minor, it's not like we'll have satellite data from before 1858, but wouldn't -infinity be the natural default here?

Reply:
Enh...  I have a slight preference for point comparisons, and 0 (1858!) does seem OK.  I don't feel strongly about it, though.